File: 백준/Bronze/17202. 핸드폰 번호 궁합/핸드폰 번호 궁합.py
a = list(input())
b = list(input())

answer = []

for i in range(len(a)):
    answer.append(a[i])
    answer.append(b[i])

temp = []


while len(answer) != 2:
    for i in range(len(answer) - 1):
        temp.append(str((int(answer[i]) + int(answer[i + 1]))%10))
    answer, temp = temp, []

print(''.join(answer))

File: 백준/Bronze/17202. 핸드폰 번호 궁합/핸드폰 번호 궁합.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? 'dev/stdin' : 'run/input.txt';
const input = fs.readFileSync(filePath).toString().trim().split('\n');

const [a, b] = input;

let answer = '';

for (let i = 0; i < a.length; i++) {
  answer += a[i] + b[i];
}

while (answer.length !== 2) {
  let temp = '';
  for (let i = 0; i < answer.length - 1; i++) {
    temp += (+answer[i] + +answer[i + 1]) % 10;
  }
  answer = temp;
}

console.log(answer);